contrib/openbsm/bin/bsmconv/bsmconv.c

```diff
@@ -0,0 +1,101 @@
+#include <sys/time.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <bsm/libbsm.h>
+#include <bsm/audit_record.h>
+
+#include "linau.h"
+#include "pjdlog.h"
+
+#define BSMCONV_BUFFER_SIZE 8192
+
+
+static void	process_events(FILE *fp);
```

Comment:
Style: there should be one space between a return type and a name of a function.